concepts/regular-expressions/about.md

```diff
@@ -137,17 +137,18 @@ sl = re.Split("12abc34(ef)", 2) // => []string{"12","(ef)"}
 sl = re.Split(" abc!", -1)      // => []string{" ","!"}
 sl = re.Split("123 456", -1)    // => []string{"123 456"}
 ```
+## Flags
 
-## Performance
+Flags change the rules that regular expression's use to search for patterns. The main flags are i for case insensitivity, m for multiline matching, s for . to match \n, and U for ungreedy or lazy matching (x* and x*?, x+ and x+? have swapped meanings). In order to set a flag in go, we include the flag preceding the regular expression pattern in the format (?x) where x is the flag.
```

Comment:
```suggestion
Flags change the rules that regular expression's use to search for patterns.
The main flags are i for case insensitivity, m for multiline matching, s for . to match \n, and U for ungreedy or lazy matching (x* and x*?, x+ and x+? have swapped meanings).
In order to set a flag in go, we include the flag preceding the regular expression pattern in the format (?x) where x is the flag.
```

Single sentence per line, rather than single line per paragraph.

Reply:
你好

Reply:
我能请教一个文体吗

Reply:
问题

Reply:
![image](https://github.com/user-attachments/assets/602355b3-9252-4182-bff6-9f4ff0980a2a)

Reply:
就是我的go环境为什么运行不了

Reply:
然后就是，现在只能靠终端的go run .运行

Reply:
> 然后就是，现在只能靠终端的go run .运行

You may want to ask for help in our Discord server or on the forms.  This is not really the place to ask about user choice configurations, such as editors or IDE's.

But when asking please do give information that describes your environment and your tools, such as your editor.

Also, if possible, please do use English, it will help get eyes on this problem as wide as possible, so that you can get the assistance you are asking for.

---

எங்கள் டிஸ்கார்ட் சர்வரில் அல்லது படிவங்களில் நீங்கள் உதவி கேட்கலாம். எடிட்டர்கள் அல்லது ஐடிஇகள் போன்ற பயனர் தேர்வு உள்ளமைவுகளைப் பற்றி கேட்க இது உண்மையில் இடம் இல்லை.

ஆனால் கேட்கும் போது, ​​உங்கள் சூழல் மற்றும் உங்கள் எடிட்டர் போன்ற உங்கள் கருவிகளை விவரிக்கும் தகவலை கொடுங்கள்.

மேலும், முடிந்தால், தயவு செய்து ஆங்கிலத்தைப் பயன்படுத்தவும், இந்த பிரச்சனையை முடிந்தவரை பரந்த அளவில் பார்க்க இது உதவும், இதன் மூலம் நீங்கள் கேட்கும் உதவியைப் பெற முடியும்.